### PR TITLE
link from Color to Colors

### PR DESCRIPTION
**Pull request by sethladd**

### CODE REVIEW

**Comment by sethladd** [comment]
Closes https://github.com/flutter/flutter/issues/953

**Comment by sethladd** [comment]
cc @abarth Hope I did this right.

File: lib/ui/painting.dart [hunk 1]
```diff
@@ -33,6 +33,10 @@ Color _scaleAlpha(Color a, double factor) {
 /// Color c1 = const Color(0xFFFFFF); // fully transparent white (invisible)
 /// Color c2 = const Color(0xFFFFFFFF); // fully opaque white (visible)
 /// ```
+///
+/// See also:
+///
+///  * [Material Design Colors](https://docs.flutter.io/flutter/material/Colors-class.html)
```
Comment (abarth) [inline lib/ui/painting.dart]: We'd usually write this as:

```
  * [Colors](...), which defines the colors found in the Material Design specification.

Comment (sethladd) [inline lib/ui/painting.dart]: Can do.

**Comment by sethladd** [comment]
ptal

**Comment by abarth** [comment]
LGTM

**Pull request merged**